tests/unit/test-class-wc-payments-features.php

```diff
@@ -160,6 +169,35 @@ function ( $pre_option, $option, $default ) {
 		$this->assertFalse( WC_Payments_Features::is_woopay_express_checkout_enabled() );
 	}
 
+	public function test_is_progressive_onboarding_enabled_returns_true() {
+		add_filter(
+			'pre_option__wcpay_feature_progressive_onboarding',
```

Comment:
Could we use the `WC_Payments_Features::PROGRESSIVE_ONBOARDING_FLAG_NAME` constant to construct the filter name? Like:

```
'pre_option_' . WC_Payments_Features::PROGRESSIVE_ONBOARDING_FLAG_NAME,
```
I think this way tests will stay more in sync with whatever changes we might make to the constant's value.